Name: Hide FileMgr Icon
Version: 2.2.4-1
Author: webOS Archive
Description: Hide the FileMgr Icon -- withour removing the app.

--- .orig/media/cryptofs/apps/usr/palm/applications/ca.canucksoftware.filemgr/appinfo.json
+++ /media/cryptofs/apps/usr/palm/applications/ca.canucksoftware.filemgr/appinfo.json
@@ -125,6 +125,7 @@
 		},
 	],
 	"copyright": "&copy; Copyright 2009-2012, Jason Robitaille",
-	"visible": true,
+	"removable": true,
+	"visible": false,
 	"uiRevision": 2
 }
